### ISSUE TITLE
What about a yaml configuration file

### ISSUE THREAD

**Issue by PurpleBabar** [squizlabs/PHP_CodeSniffer#2193]
Hi guys :)
thanks for your work on Coding Standard ;) What would you file about implementing support for yaml config files ? 

Cheerz

**Comment by gsherwood** [comment]
Additional config file formats is not something I am considering at the moment. I'm not against other formats, but I don't have the time to build and maintain them.

**Comment by PurpleBabar** [comment]
I could submit a PR if you're ok with it :)

**Comment by TomasVotruba** [comment]
@PurpleBabar Somethings like this https://github.com/Symplify/Symplify/blob/master/packages/EasyCodingStandard/config/php_codesniffer/psr2.yml#L1-L4 ? :)

**Comment by PurpleBabar** [comment]
yup :) that was what i was talking about :) Thanks for the link :)

**Comment by rask** [comment]
IMHO XML is way better than YAML. YAML is harder to read, write, and maintain.

**Comment by rodrigoprimo** [comment]
The original maintainer indicated that alternative configuration file formats were not being considered due to the maintenance burden. There has been no renewed interest in this in the new repository (PHPCSStandards/PHP_CodeSniffer) as far as I know, and the XML format is well-established.

Given the above, should this issue be closed?
